lib/msal-node/src/client/ClientCredentialClient.ts

```diff
@@ -3,24 +3,24 @@
  * Licensed under the MIT License.
  */
 
-import { ClientConfiguration } from "../config/ClientConfiguration";
-import { BaseClient } from "./BaseClient";
-import { Authority } from "../authority/Authority";
-import { RequestParameterBuilder } from "../request/RequestParameterBuilder";
-import { ScopeSet } from "../request/ScopeSet";
-import { GrantType , CredentialType, CacheOutcome, Constants, AuthenticationScheme } from "../utils/Constants";
-import { ResponseHandler } from "../response/ResponseHandler";
-import { AuthenticationResult } from "../response/AuthenticationResult";
-import { CommonClientCredentialRequest } from "../request/CommonClientCredentialRequest";
-import { CredentialFilter, CredentialCache } from "../cache/utils/CacheTypes";
-import { AccessTokenEntity } from "../cache/entities/AccessTokenEntity";
-import { TimeUtils } from "../utils/TimeUtils";
-import { StringUtils } from "../utils/StringUtils";
-import { RequestThumbprint } from "../network/RequestThumbprint";
-import { ClientAuthError } from "../error/ClientAuthError";
-import { ServerAuthorizationTokenResponse } from "../response/ServerAuthorizationTokenResponse";
-import { IAppTokenProvider } from "../config/AppTokenProvider";
-import { UrlString } from "../url/UrlString";
+import { ClientConfiguration } from "@azure/msal-common/src/config/ClientConfiguration";
```

Comment:
Are we doing `package`  based imports now?

Reply:
That is what I thought. And this is common pattern across the board, to export the files from `msal-common` and not the pre-compiled package. @konstantin-msft why the change??

Reply:
My bad. Fixed imports, thank you.